Formula/htmlq.rb

```diff
@@ -0,0 +1,18 @@
+class Htmlq < Formula
+  desc "Uses CSS selectors to extract bits content from HTML files"
+  homepage "https://github.com/mgdm/htmlq"
+  url "https://github.com/mgdm/htmlq/archive/0cb3b720451f85710b6c68894c39d521ca0d9b15.tar.gz"
```

Comment:
We only package software with tagged releases, unfortunately. Otherwise, we have no way of knowing which commit to build from when shipping this.

Reply:
Should we ask upstream to publish a release instead of using a commit artifact in here?

Reply:
Unfortunately I cannot make a release and tags. 
I am not the owner of the repository. I could make a fork, but that doesn't make because I don't develop on rust.

Reply:
You can open an issue upstream and ask the developers they'd considering making a release tag.

Reply:
Done https://github.com/mgdm/htmlq/issues/12#issuecomment-914390120. Waiting for response.